src/drive/web/modules/views/OnlyOffice/useConfig.js

```diff
@@ -2,10 +2,12 @@ import { useContext, useEffect, useState } from 'react'
 
 import { isQueryLoading } from 'cozy-client'
 import useFetchJSON from 'cozy-client/dist/hooks/useFetchJSON'
+import { generateWebLink } from 'cozy-client/dist/helpers'
```

Comment:
We should not import from `dist`. I tend to think that having a deep path is knowing the internal API of cozy-client, and we should not. 

Now, if we move the helpers files elsewhere in cozy-client, it should be tagged as a breaking change. While if we import generateWeblink from `cozy-client` we can move `helpers` where we want in cozy-client without thinking about BC.

Reply:
Exact ! Merci pour le point concernant le BC je l'avais pas en tête !